src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -87,21 +101,21 @@ export function libraryPagerGen() {
     pagerHtml = `<a href="#${activePage}" aria-label="Page ${activePage}" aria-current="true" id="va-pagination-active-num" class="va-pagination-active a-page-numbers">
     ${activePage}</a>`;
     // If we have more than one page, add a button in front of active button.
-    if (diff > 1 && (numCards === undefined || numCards > 9)) {
+    if (diff > 1 && (numCards.first === undefined || numCards.first > 9)) {
```

Comment:
In what cases would `first` or `last` be `undefined`? Not sure when these conditions would apply.

Reply:
When you aren't on the first or last pages.